extensions/amp-brightcove/0.1/validator-amp-brightcove.protoascii

```diff
@@ -36,6 +36,13 @@ tags: {  # <amp-brightcove>
     name: "data-account"
     mandatory: true
   }
+  # <amp-bind>
+  attrs: { "[data-account]" }
```

Comment:
Yes. The main purpose of this is to avoid increasing render latency or page content changing shortly after load since interpreting expressions is expensive.